<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>523</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>461</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>179</width>
       <height>459</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QComboBox" name="baudRate">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>baudRate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QComboBox" name="dataBits">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>dataBits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QComboBox" name="parity">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>parity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QComboBox" name="stopBits">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>stopBits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QComboBox" name="flowControl">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>FlowControl</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="Com_Uart">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Com_Uart</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_SetUart">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_closeUart">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>USART CONFIG</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>339</width>
       <height>219</height>
      </rect>
     </property>
     <widget class="QPushButton" name="SenSpan">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Span</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Set_My_Span">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>104</width>
        <height>87</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>339</width>
       <height>239</height>
      </rect>
     </property>
     <widget class="QPushButton" name="SetZero">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RealAngle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
